autoinstall/Pardus/preseed.cfg

```diff
@@ -16,7 +16,6 @@ d-i keyboard-configuration/xkb-keymap select us
 d-i keyboard-configuration/variant select English (US)
 
 ### Network configuration
-# Disable network configuration entirely. This is useful for cdrom
 # installations on non-networked devices where the network questions,
 # warning and long timeouts are a nuisance.
```

Comment:
The comments is incomplete. Please either make it be understandable or remove it entirely.

Reply:
remove the comments. Thanks